pkg/apis/kops/v1alpha2/instancegroup.go

```diff
@@ -121,6 +121,8 @@ type InstanceGroupSpec struct {
 	Taints []string `json:"taints,omitempty"`
 	// MixedInstancesPolicy defined a optional backing of an AWS ASG by a EC2 Fleet (AWS Only)
 	MixedInstancesPolicy *MixedInstancesPolicySpec `json:"mixedInstancesPolicy,omitempty"`
+	// CapacityRebalance configures capacity rebalancing for an AWS ASG (AWS Only)
```

Comment:
```suggestion
	CapacityRebalance makes ASGs proactively replace spot instances when the ASG receives a rebalance recommendation.
```

Reply:
Thanks for the suggestion, I've updated the comment.